crt_portal/crt_portal/settings.py

```diff
@@ -158,11 +156,12 @@
     STATIC_URL = f'https://{AWS_S3_CUSTOM_DOMAIN}/{AWS_LOCATION}/'
     STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
-
 else:
     STATIC_URL = '/static/'
 
-STATIC_ROOT = os.path.join(BASE_DIR, "static")
+STATICFILES_DIRS = ( os.path.join(BASE_DIR, 'static'), )
+MEDIA_URL = 'media/'
```

Comment:
@LindsayYoung Trying to figure out what this MEDIA_URL variable is doing — seems like it has to do with [file uploads](https://docs.djangoproject.com/en/2.2/ref/settings/#media-url). My understanding is that we don't have any features involving file uploads in the thin slice; can we safely remove this line?